helm-app/templates/api/deployment-rrd.yaml

```diff
@@ -0,0 +1,266 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}-rrd
+  labels: {{- toYaml .Values.labels.api | nindent 4 }}
+#  annotations:
+#    app.kubernetes.io/name: {{ .Values.name }}
+#    app.kubernetes.io/version: {{ .Values.version | quote }}
+#    app.kubernetes.io/owner: {{ .Values.owner }}
+#    app.kubernetes.io/env: {{ .Values.environment }}
```

Comment:
Not sure when these annotations are needed. I left it in `deployment.yaml` but commenting them out here.